src/main/java/com/blackducksoftware/integration/hub/alert/datasource/entity/GlobalConfigEntity.java

```diff
@@ -67,11 +67,14 @@
     @Column(name = "alert_digest_daily_cron")
     private String dailyDigestCron;
 
+    @Column(name = "alert_purge_data_cron")
```

Comment:
Do we have a SQL script to initialize the global config table, and has it been updated?

Reply:
We don't have an SQL script yet to initialize the tables.  I'm starting to look into that.